cli/crates/server/src/custom_resolvers.rs

```diff
@@ -165,13 +144,20 @@ async fn build_resolver(
     )
     .await?;
 
+    let process_env_prelude = format!(
```

Comment:
I see, that's how you escaped write toml with a proper library! :D

Reply:
Hehe. No, we just use a different mechanism for env variables, which is required by the DX we want